test-tools/wamr-ide/VSCode-Extension/src/extension.ts

```diff
@@ -382,10 +407,29 @@ export async function activate(context: vscode.ExtensionContext) {
             /* we should check again whether the user installed lldb, as this can be skipped during activation */
             try {
                 if (!isLLDBInstalled(context)) {
-                    await promptInstallLLDB(context);
+                    /**NOTE - if users select to skip install,
+                     *        we should return rather than continue
+                     *        the execution
+                     */
+                    if ((await promptInstallLLDB(context)) === 'skip') {
```

Comment:
Maybe we can define an enum, string union type or at least use `boolean` for the return value here and in `promptSetupDockerImages` instead of using raw strings? That'd help avoiding potential bugs in the future.